src/app/personside/infotabs/saksoversikt/saksdokumenter/DokumentListeElement.tsx

```diff
@@ -159,7 +160,7 @@ class DokumentListeElement extends React.Component<Props> {
         ]);
 
         if (!lenkeTrykket) {
-            this.visDokumentHvisTilgang(this.props.dokument.hoveddokument, this.props.dokument);
+            this.visDokumentHvisTilgang(this.props.dokumentMetadata.hoveddokument, this.props.dokumentMetadata);
```

Comment:
hm, skjønner fortsatt kke helt hvorfor du må sende inn to props her 😵

Reply:
Fordi andre steder vi bruker den funksjonen så gjør vi det på vedlegg og ikke hoveddokument, og det er en liste med dokumenter, så hvis man bare sender inn dokumentmetadata vet man ikke _hvilket_ dokument inni dokumentmetadata som skal testes på

Reply:
aight, nå skjønner jeg en gang for alle!